templates/shared/forms/interactive/managed-services.html

```diff
@@ -0,0 +1,155 @@
+                                    class="mktoField" id="utm_campaign" value="" />
+                                <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_medium"
+                                    class="mktoField" id="utm_medium" value="" />
+                                <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_source"
+                                    class="mktoField" id="utm_source" value="" />
+                                <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_content"
+                                    class="mktoField" id="utm_content" value="" />
+                                <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_term"
+                                    class="mktoField" id="utm_term" value="" />
+                                <input type="hidden" aria-hidden="true" aria-label="hidden field" name="GCLID__c"
+                                    class="mktoField" id="GCLID__c" value="" />
+                                <input type="hidden" aria-hidden="true" aria-label="hidden field" name="FBCLID__c"
+                                    class="mktoField" id="FBLID__c" value="" />
+                                <input type="hidden" aria-hidden="true" aria-label="hidden field" name="productContext"
+                                    id="product-context" value="{{ product }}" />
+                            </div>
+
+                            <div class="pagination">
+                                <a class="pagination__link--previous p-button--neutral" href="">Previous</a>
+                                <button type="submit" class="pagination__link--next p-button--positive mktoButton"
+                                    aria-label="Submit">Let's discuss</button>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="js-pagination js-pagination--4 u-hide">
+        <div class="row u-no-padding u-equal-height">
+            <div class="col-7">
+                <h3 class="p-heading--two">Thank you for enquiring about managed IT services</h3>
+                <p class="p-heading--four">A member of our team will be in touch within one working day.</p>
+            </div>
+            <div class="col-5 u-vertically-center u-align--center u-hide--small">
+                {{
+            image(
+            url="https://assets.ubuntu.com/v1/cbae9a60-thank_you_orange_cmyk.svg",
+            alt="smile",
+            width="200",
+            height="200",
+            hi_def=True,
+            loading="auto",
+            ) | safe
+          }}
+            </div>
+        </div>
+        <a class="js-close p-button--neutral" href="">Close</a>
+    </div>
+</div>
+</div>
```

Comment:
This indentation is not correct? Which `div` is it closing?

Reply:
> This indentation is not correct? Which `div` is it closing?

It should have been removed... Fixed!